dds/DCPS/XTypes/DynamicDataXcdrReadImpl.cpp

```diff
@@ -2226,6 +2226,21 @@ DDS::ReturnCode_t DynamicDataXcdrReadImpl::skip_to_struct_member(DDS::MemberDesc
         return DDS::RETCODE_NO_DATA;
       }
     }
+
+    if (member_desc->is_optional()) {
+      bool has_value = false;
+      if (!(strm_ >> ACE_InputCDR::to_boolean(has_value))) {
+        if (DCPS::DCPS_debug_level >= 1) {
+          ACE_ERROR((LM_ERROR, ACE_TEXT("(%P|%t) DynamicDataXcdrReadImpl::skip_to_struct_member: ")
+                     ACE_TEXT("Failed to read is_present for member ID %d\n"), id));
```

Comment:
```suggestion
          ACE_ERROR((LM_ERROR, "(%P|%t) DynamicDataXcdrReadImpl::skip_to_struct_member: "
                     "Failed to read is_present for member ID %d\n", id));
```
Not required, but I we might as well avoid the `ACE_TEXT` macro in new code.